scripts/rollup/build.js

```diff
@@ -235,7 +239,7 @@ function getRollupOutputOptions(
     freeze: !isProduction,
     interop: false,
     name: globalName,
-    sourcemap: false,
+    sourcemap: !isPretty,
```

Comment:
As mentioned in another comment in this review, I think maybe we should be using `shouldStayReadable` here directly, since the only time where we (currently) wouldn't be generating sourcemaps (to my knowledge) is when `prettier` is called (though personally, if it doesn't change the build time dramatically, I would be potentially inclined to keep it enabled even then)

```suggestion
    sourcemap: !shouldStayReadable,
```